hazelcast/src/main/java/com/hazelcast/client/impl/management/ManagementCenterService.java

```diff
@@ -335,4 +337,63 @@ public void shutdownMember(Member member) {
                         MCGetTimedMemberStateCodec.decodeResponse(clientMessage).timedMemberStateJson)
         );
     }
+
+    /**
+     * Checks if local MC config (client B/W list) on a given member has the same ETag as provided.
+     *
+     * @param member    target member
+     * @param eTag      ETag value of MC config to match with (should be the latest value from MC)
+     * @return          operation future object with match result: <code>true</code> if config ETags match
+     */
+    @Nonnull
+    public CompletableFuture<Boolean> matchMCConfig(Member member, String eTag) {
+        checkNotNull(member);
+        checkNotNull(eTag);
+
+        ClientInvocation invocation = new ClientInvocation(
+                client,
+                MCMatchMCConfigCodec.encodeRequest(eTag),
+                null,
+                member.getAddress()
+        );
+        return new ClientDelegatingFuture<>(
+                invocation.invoke(),
+                serializationService,
+                clientMessage -> {
+                    MCMatchMCConfigCodec.ResponseParameters response =
+                            MCMatchMCConfigCodec.decodeResponse(clientMessage);
+                    return response.response;
+                },
+                false
+        );
+    }
+
+    /**
+     * Applies the MC config (client B/W list) on a given member.
+     *
+     * @param member        target member
+     * @param eTag          ETag of the new config
+     * @param clientBwList  new config for client B/W list filtering
+     * @return              operation future object
+     */
+    @Nonnull
+    public CompletableFuture<Void> applyMCConfig(Member member, String eTag, ClientBwListDTO clientBwList) {
```

Comment:
Naming: `applyMCConfig ` --> `setMCBwListConfig`

Reply:
With current implementation the config (client filtering) itself is not stored, it's transformed and applied to all existing and new client connections. It's also not possible to obtain the config (i.e. there is no `get` operation), so that's why I'd like to avoid `set` prefix. WDYT?